draft-ietf-quic-transport.md

```diff
@@ -4491,10 +4491,9 @@ to abruptly terminate transmission on a stream.
 Receipt of a STOP_SENDING frame is invalid for a locally-initiated stream that
 has not yet been created or is in the "Ready" state (see
 {{stream-send-states}}). Receiving a STOP_SENDING frame for a locally-initiated
-stream that is "Ready" or not yet created MUST be treated as a connection error
-of type STREAM_STATE_ERROR.  An endpoint that receives a STOP_SENDING frame for
-a receive-only stream MUST terminate the connection with error
-STREAM_STATE_ERROR.
+stream that is not yet created MUST be treated as a connection error of type
```

Comment:
I think the other approach we might consider is to state that a stream enters a Ready state when the endpoints send a frame containing a higher stream ID.

Taking that approach might help resolve @mikkelfj's concern that the definition is becoming loose.

Reply:
Enters the Send state, I think you mean? Ready is the state that currently forbids STOP_SENDING.

That seems like an effective solution, although I find this use of "created" intuitive and consistent with the "Within each type, streams are created with numerically increasing stream IDs" language.